zmon-controller-app/src/main/java/org/zalando/zmon/api/EntityApi.java

```diff
@@ -71,8 +78,11 @@ public void addEntity(@RequestBody JsonNode entity) {
             if (id == null) {
                 throw new AccessDeniedException("Access denied: entity was not updated");
             }
+            String responseMessage= "Entity was successfully created with id: "+id;
+            return new ResponseEntity<>(responseMessage,HttpStatus.OK);
```

Comment:
This is changing the behavior of the API. I suggest we keep it out as we had experience before with minimal change breaking other clients.

Reply:
@mohabusama  Moreover, this code is just putting an additional message to the response which was not there previously. It doesn't change the Http status code, so the clients should be unaffected by the change as far as I can see.

Though to be on the safe side, I could double check the potential clients to verify if they would break due to this change. 

Could you please let me know which can be the potential clients? (Already tested with the ZMON-cli and it works fine)

Reply:
What is the value of the message? It adds absolutely nothing. The ID is already supplied by the user, and this basically continue in the direction of the inconsistent API responses. The fact that we need to check the clients is actually against API design, since we should assume unknown number of clients. In the end the API is consumed by a machine and this (plain/text response) is unnecessary change.